Example/Pods/CryptoSwift/Sources/CryptoSwift/AEAD/AEAD.swift

```diff
@@ -25,8 +25,8 @@ public protocol AEAD {
 
 extension AEAD {
   static func calculateAuthenticationTag(authenticator: Authenticator, cipherText: Array<UInt8>, authenticationHeader: Array<UInt8>) throws -> Array<UInt8> {
-    let headerPadding = ((16 - (authenticationHeader.count & 0xF)) & 0xF)
-    let cipherPadding = ((16 - (cipherText.count & 0xF)) & 0xF)
+    let headerPadding = ((16 - (authenticationHeader.count & 0xf)) & 0xf)
+    let cipherPadding = ((16 - (cipherText.count & 0xf)) & 0xf)
```

Comment:
swiftformat magic?
Looks like all the hexes are changed to lower characters.

Reply:
Doesn't seem so, it was an actual change in CryptoSwift:
https://github.com/krzyzanowskim/CryptoSwift/blob/master/Sources/CryptoSwift/AEAD/AEAD.swift

Reply:
Ahha yes 😅